### PR TITLE
ffmpeg (FFmpeg): update to 7.0.2

### PR DESCRIPTION
**Pull request by MingcongBai**
Topic Description
-----------------

Update FFmpeg to 7.0.2 and rebuild reverse dependencies as needed.

Package(s) Affected
-------------------

- aegisub: 3.2.2+git20231203-2
- akode: 14.1.2-2
- alsa-plugins: 1.2.12-1
- aubio: 0.4.9-4
- audacious: 4.4-4
- blender: 4.0.2-3
- chromaprint: 1.5.1-1
- cmus: 2.11.0-2
- corsixth: 1:0.67-1
- ddnet: 18.4-1
- deadbeef: 1.9.6-2
- digikam: 8.4.0-2
- dolphin-emu: 1:2407-2
- ffmpeg: 7.0.2
- ffmpegthumbnailer: 2.2.2-1
- ffmpegthumbs: 23.08.5-1
- ffms2: 5.0
- freerdp: 2:3.6.3
- gegl-0.4: 0.4.48-2
- gnome-boxes: 42.3-2
- goldendict: 1:1.5.0-1
- gpac: 2.4.0
- gstreamer: 1.24.7
- gtk-4: 4.14.2-1
- gtk-update-icon-cache: 4.14.2-1
- guvcview: 2.1.0-1
- haruna: 0.12.3-2
- harvid: 0.9.1
- k3b: 23.08.5-2
- k3b-trinity: 14.1.2-2
- k9copy-trinity: 14.1.2-1
- kfilemetadata: 5.115.0-1
- kid3: 3.9.5
- kodi: 1:21.1
- kpipewire: 5.27.11-1
- libde265: 1.0.8-4
- makemkv: 1.17.7
- mgba: 0.10.3-1
- mixxx: 2.4.1-4
- mlt: 7.26.0
- mocp: 2.6.0~svn.r3005
- moonlight-embedded: 2.5.3-2
- moonlight-qt: 4.3.0-2
- mpd: 0.23.15-6
- mplayer: 1:1.5
- mpv: 0.38.0-1
- neatvnc: 0.7.1-1
- obs-studio: 30.2.3-1
- openmw: 0.48.0+git20240902
- openscenegraph: 3:3.6.5-4
- osu-lazer: 2024.817.0-1
- owntone: 28.9-2
- pipewire: 1.2.3-1
- ppsspp: 1.17.1-1
- qmmp: 1.6.5-2
- qt-5: 1:5.15.13+webengine5.15.16+webkit5.212.0+kde20240408-6
- qt-5-doc: 1:5.15.13+webengine5.15.16+webkit5.212.0+kde20240408-6
- qt-6: 6.7.2-3
- remmina: 1.4.35
- scrcpy: 2.4-1
- simplescreenrecorder: 0.4.4
- telegram-desktop: 5.4.1
- tigervnc: 1.14.0+xserver21.1.13
- tracker-miners: 3.3.1-3
- unpaper: 1:7.0.0
- vlc: 3.0.21-3
- weston: 13.0.3
- xine-lib: 1.2.13-2
- xjadeo: 0.8.13-1
- xpra: 6.1.2

Security Update?
----------------

No

Build Order
-----------

```
#buildit ffmpeg:-pkgbreak groups/ffmpeg-rebuilds groups/ffms2-rebuilds groups/freerdp-rebuilds tigervnc ffmpeg
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by MingcongBai** [comment]
Building manually to avert unstable Internet connections on some buildbots.

**Comment by OriginCode** [comment]
Smoke tests on amd64:
blender
圖片
mixxx
圖片
guvcview
圖片
scrcpy
圖片
unpaper (I don't have scanned files)
圖片
corsixth
圖片
ddnet
圖片
dolphin-emu
圖片
mgba
圖片
osu-lazer
圖片
ppsspp
圖片
goldendict
圖片
aubio (No valid test files)
圖片
cmus
圖片
deadbeef
圖片
ffmpegthumbnailer
圖片
aegisub **SIGSEGV**
圖片
gpac **Black screen on amd64, SIGSEGV on loongarch64**
harvid (No test env)
圖片
kid3
圖片

**Comment by OriginCode** [comment]
makemkv (**PENDING DROP, ORPHANING**)
圖片
mocp
圖片
moonlight-embedded
圖片
moonlight-qt
圖片
mpd
圖片
mpv
圖片
obs-studio
圖片
owntone
圖片
pipewire: working
qmmp
圖片

**Comment by OriginCode** [comment]
simplescreenrecorder, haruna
Screenshot_20240919_015857

**Comment by OriginCode** [comment]
vlc
圖片
xjadeo
圖片
freerdp
圖片
remmina
圖片
gnome-boxes (**SIGTRAP, conflicting symbols**)
圖片
xpra (**Need copy-repo**)
圖片
telegram-desktop
圖片
tracker-miners (**No test env**)

**Comment by OriginCode** [comment]
k3b
圖片
digikam
圖片
k3b-trinity
圖片
k9copy-trinity
圖片
synfig (**Missing reworked package**)
openmw (**Installs 3rd party software, and fails to do so**)
圖片
圖片
kodi (**Black screen when playing video, audio works**)
圖片
audacious
圖片
mplayer
圖片
hmcl
圖片
openscenegraph
圖片
tigervnc (**Missing font?**)
圖片

**Comment by MingcongBai** [comment]
>  tigervnc (Missing font?)

@OriginCode `vncviewer` is fine here:

图片

**Comment by MingcongBai** [comment]
Fixed OpenMW:

图片

`gnome-boxes` and `makemkv` has been dropped.

**Comment by MingcongBai** [comment]
@OriginCode KODI works well for me here (Wayland issue?):

图片

**Comment by MingcongBai** [comment]
Neither `aegisub` nor `gpac` segfault'd on AMD64, but the latter did segfault on LoongArch. Hmm.

**Comment by OriginCode** [comment]
> Neither `aegisub` nor `gpac` segfault'd on AMD64, but the latter did segfault on LoongArch. Hmm.

`aegisub` does not work with native Wayland, upstream issue https://github.com/Aegisub/Aegisub/issues/180
Use `GDK_BACKEND=x11` to force it use X11 as a workaround.

**Comment by OriginCode** [comment]
> @OriginCode KODI works well for me here (Wayland issue?):
> 
> 图片

Workaround supplying `--viewing=X11` to force it using X11 on Wayland works.

**Comment by MingcongBai** [comment]
Dropping gpac (and ogmrip as a result), the latter is barely maintained and is the sole dependee.

This topic should be good to go. cc @OriginCode

File: app-multimedia/ffmpegthumbnailer/autobuild/defines [hunk 1]
```diff
@@ -3,7 +3,6 @@ PKGSEC=utils
 PKGDEP="gvfs ffmpeg libjpeg-turbo libpng"
 PKGDES="Lightweight video thumbnailer that can be used by file managers"
 
-ABTYPE=cmake
```
Comment (OriginCode) [inline app-multimedia/ffmpegthumbnailer/autobuild/defines:6]: Why removing `ABTYPE`?

**Review by OriginCode — CHANGES_REQUESTED** [review]

File: app-multimedia/ffmpegthumbnailer/autobuild/defines (hunk 1 above)
Comment (MingcongBai) [reply app-multimedia/ffmpegthumbnailer/autobuild/defines:6]: Fixed. I meant to specify `cmakeninja`.

**Review by OriginCode — APPROVED** [review]

**Review by jiegec — COMMENTED** [review]
gnome-remote-desktop needs to be removed from gnome-base as well.

**Pull request merged**